### PR TITLE
chore(codebuild): add cfn and codebuild files for release

### PR DESCRIPTION
**Pull request by RitvikKapila**
_Issue #, if available:_

_Description of changes:_

_Squash/merge commit message, if applicable:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cfn/python/MPL-Python.yml [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+            - BUILD_GENERAL1_LARGE
+        TimeoutInMins: 480
+      LogsConfig:
+        CloudWatchLogs:
+          Status: ENABLED
+        S3Logs:
+          Status: DISABLED
+          EncryptionDisabled: false
+
+  # CodeBuildProjectProdRelease:
```
Comment (lucasmcdonald3) [inline cfn/python/MPL-Python.yml:132]: Why are prod release resources commented out? "Safety"?

**Review by lucasmcdonald3 — COMMENTED** [review]
This is fine.
We are accepting that this isn't tested and we can't test this because we don't want to accidentally release code as part of testing this.
We'll test as part of release.

File: cfn/python/MPL-Python.yml (hunk 1 above)
Comment (RitvikKapila) [reply cfn/python/MPL-Python.yml:132]: Yes, can uncomment them. These are just resources.

File: cfn/python/MPL-Python.yml [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+    ParameterGroups:
+      - Label:
+          default: Crypto Tools CodeBuild Project Template
+        Parameters:
+          - ProjectName
+          - ProjectDescription
+          - SourceLocation
+
+Resources:
+  # CodeBuildProject:
```
[the remark this answers is not in the document]
Comment (RitvikKapila) [reply cfn/python/MPL-Python.yml:39]: okay, resolved. Thanks

**Pull request closed without merging**